index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <img src="images.svg\image_geometry_1 (1).svg" alt="" class="top-left">
+        <img src="images.svg\ic_menu (1).svg" alt="" class="hamburger">
+    </header>
+    <main>
+        <section class="section-1">
+            <h1>Iheagwara<br>Augustine</h1>
```

Comment:
- [ ]  I can see that all is good with your project but you forget to put  I'm Iheagwra ,... kindly put this to be more specific on your identity name :wink:  - [ ]

Reply:
i don't understand this , in the template there is no "I'm" please clarify

Reply:
> * [ ]   I can see that all is good with your project but you forget to put  I'm Iheagwra ,... kindly put this to be more specific on your identity name 😉  - [ ]

i don't understand this , in the template there is no "I'm" please clarify

Reply:
- [ ] I can see that you did not take care of this kindly fix thsi , it's better to introduce yourself in a profesional way so that recruiters can be attractive and reach you out , try to make it profesional , you are almost there , you can make it i believe so :upside_down_face:

Reply:
> * [ ]  I can see that you did not take care of this kindly fix thsi , it's better to introduce yourself in a profesional way so that recruiters can be attractive and reach you out , try to make it profesional , you are almost there , you can make it i believe so 🙃

please what do you mean? where do i introduce myself i did exactly what was on the template

Reply:
.

Reply:
- [ ] Hey @stino-x  that is an optional review , you should focus on the request changes given , I told you that you should use `nav` tag for you hambuger menu instead of putting them in `img` tag that was not required for this project  I encourage you to follow the requirements for this project thanks. :wink: